src/babel-plugin-transform-sx-tailwind/src/__tests__/fixtures/template-literal-expression/output.js

```diff
@@ -0,0 +1,18 @@
+// @flow
+import React, { type Node } from 'react';
+import * as sx from '@adeira/sx';
+export default function Example(): Node {
+  return <div className={`${__styles_2aHSqM('px-4')} ${styles('customStyle')}`}>Lorem lipsum</div>;
```

Comment:
https://github.com/adeira/universe/pull/1289

Reply:
Valid point, but this is supposed to be used as a hack when you find something is missing in the SX Tailwind. This was you can locally add missing CSS until it will be fixed in the library.

Reply:
Understand, it's your call. Just pointing out that the resulting style is a bit unknown and depends on external factors (order of stylesheets in this case). 🙂